tests/e2e/helm_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/yaml"
+)
+
+// HelmReleaseStatus describes the possible states of a Helm release.
+type HelmReleaseStatus string
+
+const (
+	// HelmReleaseDeployed is the Helm release state where the deployment is
+	// successfully applied to the cluster.
+	HelmReleaseDeployed HelmReleaseStatus = "deployed"
+
+	// HelmReleaseFailed is the Helm release state where the deployment
+	// encountered an error and couldn't be applied successfully to the cluster.
+	HelmReleaseFailed HelmReleaseStatus = "failed"
+)
+
+// HelmRelease describes a Helm
+type HelmRelease struct {
+	ReleaseName string            `json:"name" yaml:"name"`
+	Namespace   string            `json:"namespace" yaml:"namespace"`
+	Revision    string            `json:"revision" yaml:"revision"`
+	UpdatedTime string            `json:"updated" yaml:"updated"` // Note: not parsable implicitly.
+	Status      HelmReleaseStatus `json:"status" yaml:"status"`
+	Chart       string            `json:"chart" yaml:"chart"`
+	AppVersion  string            `json:"app_version" yaml:"app_version"`
+}
+
+// installHelmChart deploys a Helm chart to the specified kubectl context and
+// namespace using the specified infos, extra arguments can be any of the helm
```

Comment:
In the description I can see "extra arguments can be any of the helm CLI install flag arguments" but the arguments list does not contain the extra arguments for CLI install flag.

Reply:
Fixed in 5bfcc05.